test/browser_tests/spec_suites/wagtail/wagtail-page-states-pages.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var DraftPage = require(
+  '../../page_objects/page_wagtail_states_pages.js' ).draftpage;
+var SharedPage = require(
+  '../../page_objects/page_wagtail_states_pages.js' ).sharedpage;
+var LivePage = require(
+  '../../page_objects/page_wagtail_states_pages.js' ).livepage;
+var LiveDraftPage = require(
+  '../../page_objects/page_wagtail_states_pages.js' ).livedraftpage;
+
+describe( 'Wagtail Draft Page', function() {
+  var page;
+
+  beforeAll( function() {
+    page = new DraftPage();
+    page.get();
+  } );
+
+  it( 'should not load in a browser',
+    function() {
+      expect( 404 );
```

Comment:
I don't think this'll do what you expect. It's checking whether the number 404 is 404, which it always will be true. You can change the number here and it'll still pass.

Looks like checking response headers is complicated http://stackoverflow.com/questions/25137881/how-to-use-protractor-to-get-the-response-status-code-and-response-text It might be easier to find something unique in the 404 pages and check for that.